src/query.md

```diff
@@ -3,7 +3,7 @@
 <!-- toc -->
 
 As described in [the high-level overview of the compiler][hl], the Rust compiler
-is still (as of <!-- date-check --> July 2021) transitioning from a
+is still (as of <!-- date-check --> April 2023) transitioning from a
```

Comment:
Yes. Right now the resolver is just one giant "all_resolutions" query (instead of a separate query invocation per-name), and rustc_ast_lowering isn't incremental at all to my knowledge.